buildSrc/src/test/java/com/osacky/flank/gradle/integration/SanityRoboCheck.kt

```diff
@@ -0,0 +1,328 @@
+package com.osacky.flank.gradle.integration
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+class SanityRoboCheck {
+  @get:Rule
+  var testProjectRoot = TemporaryFolder()
+
+  @Before
+  fun setUp() = testProjectRoot.newFile("flank-gradle-service.json").writeText("{}")
+
+  @Test
+  fun checkSanityRoboRunSimpleCase() {
```

Comment:
instead of simple case, can we be more specific?
```suggestion
  fun checkSanityRoboRunWithProjectProperty() {
```